### PR TITLE
The Newsweek Project: Milestone 1

### PR DESCRIPTION
**Pull request by Emmyn5600**
This is a clone of The Newsweek Website homepage using HTML & CSS & bootstrap: https://web.archive.org/web/20210120125445/https://www.newsweek.com/ .

Moreover, This pull request contains the following changes:

- Added index.html file
- Added style.css file
- Added Header and Footer sections
- Worked on the responsiveness with 3 different screens

Best,

Emmy & Maha

### CODE REVIEW

File: index.html
Comment (teekaytech) [inline index.html]: The `Login` and `Subscribe` should have links (i.e. `<a>` tag).

File: index.html
Comment (teekaytech) [inline index.html]: The Newsweek logo should be a link

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @Emmyn5600  :wave:

__You've done an awesome job in this milestone :clap:__

- Code linters are running successfully :+1: 
- Page looks clean so far :ok: 
- Nice PR title and summary :1st_place_medal: 

__However, some changes are required before this milestone can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] The `Issues page` and `MIT license` links in your Readme are broken. 
- [ ] You're overusing the `!important` rule. Believe me, you can do completely without the `!important` rule because it will end up messing up your cascading rules. Besides, it's one of the [common mistakes in CSS style](https://speckyboy.com/good-bad-css-practices/) that you should avoid.
- [ ] Please check my inline comments below.

**[ OPTIONAL CHANGES ]**

- [ ] You might want to remove the left and right spaces of the entire page on tablet view. This behavior can be observed on the original page.
**Clone page**
ddkdkd

**Original page**
clone

* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by smcommits — CHANGES_REQUESTED** [review]
# Status: Changes Required

### Hello, team! Excellent job. Your project looks great. I can see that you have implemented almost all the changes requested by the previous TSE. But, you still have not fixed one issue that was pointed out by the previous TSE. 

## Required Changes

**You haven't implemented the following change request by the previous TSE.** 

- [ ] **Limiting the use `!important` in your CSS.**
You still have a lot of "!important" rules in your CSS. This is a very important thing to consider. Using the !important declaration is often considered bad practice because it has side effects that mess with one of CSS's core mechanisms: specificity. In many cases, using it could indicate poor CSS architecture.

For instance,  consider the following selector from you CSS file 
`.font-hover-black:hover {
  color: black !important;
}`

You don't need important here as there is not any previous selector which targets this element with higher specificity. You can remove important and it will still work. 

Please implement the changes and request another review from your dashboard. 

# 
### If you need any help, or you think I have failed to notice something and have made an unreasonable request, please don't hesitate to reach out to me on [slack](https://microverse-students.slack.com/team/U017N7LLZ40)

**Review by rukundoeric — APPROVED** [review]
# Status: 🍾 PROJECT APPROVED 🍾
_Excellent job on this project guys. 🎆 Keep it up._

- ### To Highlight
  - [x] The project is clean. 💯
  - [x] The project follows Git flow. 💯
  - [x] Nice job implementing all feedback.  💯

- ### To Improve
  - N/A.

_Congratulations on the splendid job **@team** 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/) ,
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/),
Slack: Rukundo Eric**

**Pull request merged**
